frontend/app/test/unit/test-utils.ts

```diff
@@ -52,18 +70,11 @@ export async function userTypeInputs(user: UserEvent, inputs: { [key: string]: s
   }
 }
 
-export const setupTestRouter = () => {
-  return createMemoryRouter(routes, {
-    future: {
-      v7_normalizeFormMethod: true,
-    },
+export function getCandidateFullNamesFromMockData(politicalGroupMockData: PoliticalGroup): string[] {
+  const candidateNames = politicalGroupMockData.candidates.map((candidate) => {
+    return candidate.first_name
+      ? `${candidate.last_name}, ${candidate.initials} (${candidate.first_name})`
+      : `${candidate.last_name}, ${candidate.initials}`;
```

Comment:
might want a formal name rendering utility for this.

Reply:
Not sure what you mean by "fornal name rendering utility"? Goal of this function is to concatenate the relevant parts of the mock data into the string shown on the screen. The simpler we keep that function the better.